src/main/java/org/cascadebot/cascadebot/commands/music/LoopCommand.java

```diff
@@ -0,0 +1,59 @@
+/*
+
+  * Copyright (c) 2019 CascadeBot. All rights reserved.
+  * Licensed under the MIT license.
+
+ */
+
+package org.cascadebot.cascadebot.commands.music;
+
+import net.dv8tion.jda.core.entities.Member;
+import org.cascadebot.cascadebot.commandmeta.*;
+import org.cascadebot.cascadebot.commandmeta.Module;
+import org.cascadebot.cascadebot.permissions.CascadePermission;
+
+import java.util.Set;
+
+import static org.cascadebot.cascadebot.music.CascadePlayer.LoopMode.*;
```

Comment:
I know binary is fixing this, but WHY ARE YOU STILL USING STATIC!

Reply:
should i just use import instead?